src/io/probst/idea/clangformat/ClangFormatAction.java

```diff
@@ -65,19 +69,12 @@ public void actionPerformed(AnActionEvent actionEvent) {
     // clang-format.
     int docLength = document.getTextLength() - 1;
     int cursor = Math.min(caret.getOffset(), docLength);
-    int selectionStart = Math.min(editor.getSelectionModel().getSelectionStart(), docLength);
```

Comment:
with this change, you can no longer change an individual selection range, can you?